README.md

```diff
@@ -20,6 +20,8 @@ Please do not use this code in production, until code goes through more testing
 
 ###Building and Testing
 
+**Note:** Vagrant 1.7.4 and VirtualBox 5.0+ are required to build and test netplugin.
```

Comment:
Let's wait for him to answer -- I think he said he was going on vacation. We can always merge that after the fact.